Test/Graph/test_graph.py

```diff
@@ -244,17 +244,6 @@ def test_grid_color_pbc():
     assert sorted(g1.edges()) == sorted(g2.edges())
 
 
-def test_automorphisms():
-    for graph in lattices:
-        if graph.is_connected():  # to avoid troubles with ig automorphisms
-            g = ig.Graph(edges=graph.edges())
```

Comment:
I think we can actually completely remove the dependence on igraph even in the test dependencies (and in this file's import) ?